acceptance-tests/src/test/resources/uk/gov/di/test/features/Strategic_App.feature

```diff
@@ -22,18 +22,10 @@ Feature: Sign in lockouts for Strategic App
     Then the user is taken to the "Enter your email" page
     When the user enters their email address
     Then the "You cannot sign in at the moment" lockout screen is displayed
-    And the "You can also wait 2 hours, then try again." retry Text is displayed
+    And the "[Reset your password, ., , You can also wait 2 hours, then try again.]" retry Text is displayed
```

Comment:
This is a brittle way to test this page unless its important that some of the text appears on its own line.  If that is important then it would be better to have multiple tagged lines.  Consider normalising the text by removing any formatting or use a doc string to represent the expected text.